build-ps/ubuntu/rules

```diff
@@ -218,22 +218,11 @@ override_dh_strip:
 
 override_dh_installinit:
 	@echo "RULES.$@"
-<<<<<<< HEAD
 	if [ "$(DISTRELEASE)" != "precise" -a "$(DISTRELEASE)" != "trusty" -a "$(DISTRELEASE)" != "wheezy" ]; then dh_systemd_enable --name=mysql; fi
```

Comment:
This doesn't look good, now we enable the mysql service at two places, in one of them with a condition, in another unconditionally. It's probably better to keep ours or to ask @EvgeniyPatlan about how to resolve it.

Reply:
Fixed after consultation with @EvgeniyPatlan